### ISSUE TITLE
`--fail-level` for "The following cops were added to RuboCop"

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#8246]
## Is your feature request related to a problem? Please describe.

Hello.

It'd be easier to track CI builds, especially from Dependabot or Depfu ones, when there are fails for warnings about new cops like this:

```
> bundle exec rubocop
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Style/RedundantFetchBlock (0.86)
For more information: https://docs.rubocop.org/rubocop/versioning.html
Inspecting 7 files
.......
```

But no one value of `--fail-level` option makes this fail.

## Describe the solution you'd like

Add a new value for `--fail-level` which will fail runs with warnings about new cops.

## Describe alternatives you've considered

I've tried all existing `--fail-level` values.

**Comment by bbatsov** [comment]
That's a purely informational message (not a warning) that everyone can safely disregard. Why should it result in a failure under any conditions?

**Comment by AlexWayfer** [comment]
> Why should it result in a failure under any conditions?

As I've described, it'd be helpful for CI builds. It's just an option anyway. So, when I'm getting PRs from bots now — checks are successful, but I want to know that there were added new cops and enable them before merging. But I have to check the output manually. Especially with third-party CIs there are extra clicks.

**Comment by AlexWayfer** [comment]
Or maybe we can add a configuration option like:

```yaml
NewCops:
  Enabled: true
```

`NewCops` is a group, like `AllCops`, but contains all undecided cops (about which there are warnings).

**Comment by bbatsov** [comment]
Well, this already exists - see https://docs.rubocop.org/rubocop/0.86/versioning.html#pending-cops :wink:

**Comment by AlexWayfer** [comment]
> Well, this already exists - see https://docs.rubocop.org/rubocop/0.86/versioning.html#pending-cops

Oh… thank you. My bad. I'll use it and going to close this issue.

**Comment by AlexWayfer** [comment]
> Well, this already exists - see https://docs.rubocop.org/rubocop/0.86/versioning.html#pending-cops

I already did mistake with `enabled` and `enable`: there is `Enabled: true/false` for cops and `NewCops: enable` (not `enabled`). 🤯
